react/ui-components/src/atoms/BoundaryFilter.js

```diff
@@ -0,0 +1,569 @@
+import React, { useEffect, useState } from 'react'
+import { Card, CardText, MultiSelectDropdown, Dropdown, Toast } from '@egovernments/digit-ui-components'
```

Comment:
I can see 2 BoundaryFilter components, @abishekTa-egov you need to ensure from your side that your PR is cleaned up before you send for approval. Don't waste other people's time please